app/Http/Requests/Rdt/RdtApplicantUpdateRequest.php

```diff
@@ -45,4 +47,24 @@ public function rules()
             'symptoms_activity'    => ['sometimes', 'required'],
         ];
     }
+
+    private function ruleNik()
+    {
+        $rule = ['sometimes', 'required'];
+        if ($this->nationality == NationalityEnum::WNI()) {
```

Comment:
```suggestion
        $nik = array_merge($rule, ['regex:/^[A-Z0-9]{7,9}$/']); // validation passport, min:7, max:9 character
        if ($this->nationality == NationalityEnum::WNI()) {
            $nik = array_merge($rule, [new NikRule()]); // validation nik
        }
```

kalo dibuat seperti ini gmana kang? masuk ga ya?

Reply:
masuk kang